pyscal/pyscallist.py

```diff
@@ -338,7 +341,6 @@ def make_ecl_output(
         self,
         keyword: str,
         write_to_filename: Optional[str] = None,
-        gaswater: bool = False,
```

Comment:
What changed?

Reply:
Nothing. This option is a leftover from the early implementation that never got used.